### PR TITLE
updating train_reader_extractive.sh  to use train_seq_2_seq.py

### PR DESCRIPTION
**Pull request by atnafuatx**

### CODE REVIEW

**Comment by ToluClassics** [comment]
Hi @atnafuatx, can you create a new file instead of modifying the existing one

maybe you can name it `train_reader_generative.sh`?

**Comment by atnafuatx** [comment]
@ToluClassics  ok

**Comment by atnafuatx** [comment]
@ToluClassics  check now

File: scripts/train_reader_generative.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+model_name_or_path=ToluClassics/extractive_reader_nq
```
Comment (ToluClassics) [inline scripts/train_reader_generative.sh]: need to change to mt5-base

File: scripts/train_reader_generative.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  --output_dir $output_dir/$model_name_or_path \
+  --save_steps $save_steps \
+  --overwrite_output_dir \
+  --push_to_hub \
+  --push_to_hub_model_id=extractive_reader_nq_squad_v2 \
+  --weight_decay 0.01 \
+  --eval_steps 1000 \
+  --logging_steps 1000 \
+  --evaluation_strategy="steps" \
+  --version_2_with_negative
```
Comment (ToluClassics) [inline scripts/train_reader_generative.sh:33]: we should probably add these parameters too 

```
  --context_column context \
  --question_column question \
  --answer_column answers \
```

File: scripts/train_reader_extractive.sh [hunk 3]
```diff
@@ -7,7 +7,7 @@ num_train_epochs=5
 max_seq_length=384
 save_steps=10000
 
-CUDA_VISIBLE_DEVICES=4 python3 baselines/reader/train_extractive.py \
+CUDA_VISIBLE_DEVICES=4 python3 baselines/reader/train_seq_2_seq.py \
```
Comment (ToluClassics) [inline scripts/train_reader_extractive.sh]: can we restore this file to it's previous state?

Comment (ToluClassics) [reply scripts/train_reader_extractive.sh]: It shouldn't be in this PR

**Review by ToluClassics — APPROVED** [review]

File: scripts/train_reader_extractive.sh (hunk 3 above)
Comment (atnafuatx) [reply scripts/train_reader_extractive.sh]: changed

File: scripts/train_reader_generative.sh (hunk 1 above)
Comment (atnafuatx) [reply scripts/train_reader_generative.sh:1]: where?

**Pull request merged**